mock/server.py

```diff
@@ -7,22 +7,30 @@
 #
 # Copyright Contributors to the Zowe Project.
 #
-
+import os
 from flask import Flask
 from flask import request
 import base64
 from flask import make_response
-
+import math
 app = Flask(__name__)
 
+
+
 ###  TODO:
 #RBAC routes for permissions
 #Permissions changed to array [7,7,7] to better represent all the different types of permissoins and add permission check function
 
 global_username = "mock"
 global_password = "pass"
 global_password_expired = False
-
+#set port number that server uses,could be change to any number
+os.environ['FLASK_RUN_PORT'] = '5000' 
```

Comment:
We don't want to set it to 5000, instead we want to default it to 5000 only if the user env variable isn't specified (because we want the server to respect an env varialbe) so instead of
`os.environ['FLASK_RUN_PORT'] = '5000'
...
global_port = os.getenv('FLASK_RUN_PORT')`

we want something like
`global_port = os.getenv('FLASK_RUN_PORT') if os.getenv('FLASK_RUN_PORT') != undefined else '5000'`